digital_asset_types/src/dao/mod.rs

```diff
@@ -93,6 +96,23 @@ impl SearchAssetsQuery {
                     .clone()
                     .map(|x| asset::Column::SpecificationAssetClass.eq(x)),
             )
+            .add_option(
```

Comment:
i guess we can remove specification_asset_class option now as we have token_type

Reply:
yeah think so.